vars/odsOrchestrationPipeline.groovy

```diff
@@ -82,7 +84,17 @@ def call(Map config) {
         withPodTemplate(odsImageTag, steps, alwaysPullImage) {
             logger.debugClocked('pod-template')
             withEnv (envs) {
-                def result = new InitStage(this, project, repos, startAgentStage).execute()
+                def result
+                def cannotContinueAsHasOpenIssuesInClosingRelease = false
+                try {
+                    result = new InitStage(this, project, repos, startAgentStage).execute()
+                } catch (OpenIssuesException ex) {
+                    cannotContinueAsHasOpenIssuesInClosingRelease = true
+                }
+                if (cannotContinueAsHasOpenIssuesInClosingRelease) {
+                    logger.warn('Cannot continue as it has open issues in the release.')
+                    return
+                }
```

Comment:
@jorge-romero why rather complex? wouldn't the logger.warn + return inside the catch block be enough?

Reply:
That's was my first implementation, but it was complaining about trying to finalize the pipeline within a catch block, that's why I had to rewrite in this ugly way unfortunally

Reply:
Another solution may be logger.warn + result=false inside the catch block. And remove cannotContinueAsHasOpenIssuesInClosingRelease var.